### DISCUSSION TITLE
YN0013 can't be found 404

### DISCUSSION THREAD

**Discussion by luismendes070** [npm/feedback#176] [CLI] [answered]
https://github.com/babel/babel/tree/master/packages/babel-plugin-syntax-dynamic-import

**Comment by darcyclarke** [comment, marked as answer]
This doesn't look like an npm specific piece of feedback, rather a yarn error status code. `YN00013` seems to represent a "fetch not cached" ~ https://yarnpkg.com/advanced/error-codes#yn0013---fetch_not_cached

**Reply by arcanis** [reply]
Yeah, it points to an error that happened during a remote fetch - given the issue title, I guess OP received a 404 error. Given the lack of details, I suspect the problem comes from the dependencies they list (probably a typo) rather than the registry itself.
